rtc/troubleshooting.md

```diff
@@ -29,8 +29,36 @@ As a result of this process:
 
 Care must be taken to avoid losing encryption keys to this race condition.
 
-### Server reports JWT authentication was issued in the future
+## Common JWT error messages
 
-JWT is a very strict protocol. If the computer that signs the JWT has a clock that is significantly ahead of the server clock, and tokens are signed on demand, this error can occur.
+If the below error descriptions do not immediately solve the problem, try pasting the token into [https://jwt.io/](https://jwt.io/) which will provide more detailed token format debugging.
+
+### Unsupported JWT signature algorithm
+
+RTC requires an [asymmetric signing algorithm]({{site.baseurl}}/rtc/jwt-authentication/#supportedalgorithms) for JWT identity tokens. Asymmetric signatures use separate private and public keys; the public key stored on the {{site.cloudname}} server can only be used to verify signatures, not create new ones. The private key, used to create new signatures, remains private.
```

Comment:
That seems bad for SEO 🤔

Reply:
I do that when the link covers a lot more than the topic, but here the link is _specifically_ about the asymmetric signing algorithms we support 🤔

Reply:
Why not `For more information on the asymmetric signing algorithms we support, see [Name of Page — Subheading](...)`?

Reply:
`more` is unnecessary, `we` is unclear, and that sentence is written in passive voice: https://www.tiny.cloud/docs/configure/contributing-docs/#useactivevoicenotpassivevoice

Reply:
I did this on [the other page too](https://github.com/tinymce/tinymce-docs/pull/1916/files?file-filters%5B%5D=.md#diff-2bcdccd2556c30529dbafd69d004d4b74d1f09e042a9daf649333c4426e47d15R21), and it wasn't picked up. I think linking mid-sentence is better UX.

Reply:
not really. I did see this in the very technical part but ignored it because it's long with too many links to pull out. screen readers will get mid-sentence, then read out a URL, so bad UX

Reply:
If we used relative links, it would probably be fine. The way we do it is called out as bad as well.
https://usability.yale.edu/web-accessibility/articles/links#link-text

Reply:
yeah, that's about people doing this: 
![image](https://user-images.githubusercontent.com/14833627/114962922-fbbe8f80-9eae-11eb-9936-5d7f4561706d.png)
which falls under my first point here: https://www.tiny.cloud/docs/configure/contributing-docs/#linksandcross-references

Reply:
I think it applies to "see: <link text>" as well, but ok.